### PR TITLE
CAKE-5359 | Style Affiliate Unit

### PR DESCRIPTION
**Pull request by cschucode**
This PR included the addition of affiliate units and assets [CAKE-5374]
- Added affiliate units for Disney+ and DDB
- Style Units per Zeplin
- Using theming for background-color, color, and logos

[CAKE-5374]: https://wikia-inc.atlassian.net/browse/CAKE-5374

### CODE REVIEW

File: extensions/wikia/AffiliateService/styles/affiliate-unit.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+        @if $is-dark-wiki {
+            border: solid 1px rgba($color-page, 0.5);
+        }
+    }
+
+    &__image-wrapper {
+        background-repeat: no-repeat;
+        background-size: cover;
+        clip-path: polygon(0% 0%,100% 0%,80% 100%,0 100%);
```
Comment (vforge) [inline extensions/wikia/AffiliateService/styles/affiliate-unit.scss]: It's not even supported in edge https://caniuse.com/#feat=css-clip-path :(

Comment (vforge) [reply extensions/wikia/AffiliateService/styles/affiliate-unit.scss]: Was the browser fallback been accepted by Design?

**Review by vforge — CHANGES_REQUESTED** [review]

File: extensions/wikia/AffiliateService/styles/affiliate-unit.scss (hunk 1 above)
Comment (cschucode) [reply extensions/wikia/AffiliateService/styles/affiliate-unit.scss]: No. Only tested in Chrome so far. Doesn't work on Safari either. Will look for another method.

**Comment by mklucsarits** [comment]
Looks good to me. Are we putting the console logging on prod for debugging reasons?

**Comment by cschucode** [comment]
TBH... I was letting it hang out just in case. I'll remove it for now and add back as needed.

**Pull request merged**